server/src/db/system_tables.rs

```diff
@@ -1,20 +1,27 @@
-use std::any::Any;
+//! Contains implementation of IOx name: (), stats: () system tabl stats: () stats: ()es (aka tables in the `system` schema)
```

Comment:
I'm... not sure what this comment is saying (especially `stats: ()es` and what `()` means in general), I might be missing some database lingo. I think there's at least an e missing though?

```suggestion
//! Contains implementation of IOx name: (), stats: () system table stats: () stats: ()es (aka tables in the `system` schema)
```